integrations/gitlab/gitlab_integration/gitlab_service.py

```diff
@@ -535,6 +549,77 @@ async def get_all_issues(self, group: Group) -> typing.AsyncIterator[List[Issue]
             issues: List[Issue] = typing.cast(List[Issue], issues_batch)
             yield issues
 
+    async def get_all_group_members(
+        self, group: Group
+    ) -> typing.AsyncIterator[List[GroupMember]]:
+
+        try:
+            port_app_config: GitlabPortAppConfig = typing.cast(
+                "GitlabPortAppConfig", event.port_app_config
+            )
+            filter_bots = port_app_config.filter_bots
+
+            def skip_validation(_: User) -> bool:
+                return True
+
+            def should_run_for_member(member: GroupMember) -> bool:
+                return not member.username.__contains__("bot")
+
+            validation_func: Union[
+                Callable[[User], bool], Callable[[GroupMember], bool]
+            ] = (should_run_for_member if filter_bots else skip_validation)
+
+            logger.info(f"Fetching all members of group {group.name}")
+            async for members_batch in AsyncFetcher.fetch_batch(
+                fetch_func=group.members.list,
```

Comment:
why `group.member.list` and not `group.member_all.list` ?
https://python-gitlab.readthedocs.io/en/stable/gl_objects/groups.html#id10 
what do you think about adding this as an option to query all rather than only in one hierarchy?